### PR TITLE
CDS-48216-jira-issuelink-support

### PR DESCRIPTION
**Pull request by michaelcretzman**
https://harness.atlassian.net/browse/CDS-48216

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ x] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* https://harness.atlassian.net/browse/CDS-48216

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by aimurphy — DISMISSED** [review]

File: docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-jira-issues-in-cd-stages.md [hunk 1]
```diff
@@ -100,3 +100,23 @@ Locate **URL** and copy its value.
 In a new browser tab, paste the URL and press enter.
 
 The new issue appears in Jira.
+
+## Issue link support
```
Comment (naman-goenka) [inline docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-jira-issues-in-cd-stages.md:104]: "issuelink" is a field type in Jira, and supporting that is a technical detail. For documentating this feature, we can simply mention `Parent` field support and an example for create and update steps.

File: docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-jira-issues-in-cd-stages.md [hunk 2]
```diff
@@ -100,3 +100,23 @@ Locate **URL** and copy its value.
...
+
+Harness supports issue links (the `issuelink` field in Jira).
```
Comment (naman-goenka) [inline docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-jira-issues-in-cd-stages.md:106]: `issuelink` is not a field but a field type we can omit it in docs

File: docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-jira-issues-in-cd-stages.md [hunk 3]
```diff
@@ -100,3 +100,23 @@ Locate **URL** and copy its value.
...
+## Issue link support
+
+Harness supports issue links (the `issuelink` field in Jira). 
+
+This support enables you to do the following:
+
+* The Jira Create step can now be used to create issues with existing issues as their parent.
+* The Jira Update step can be used to update the parent of a specific issue.
+* A sub-task can be created using the Jira Create step.
+* A ticket parent can be provided by simply typing the parent issue key in **Parent Link** setting, such as `TJI-47890`.
```
Comment (naman-goenka) [inline docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-jira-issues-in-cd-stages.md:113]: field name is `Parent` not `Parent Link`

File: docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-jira-issues-in-cd-stages.md [hunk 4]
```diff
@@ -100,3 +100,23 @@ Locate **URL** and copy its value.
...
+Harness supports issue links (the `issuelink` field in Jira). 
+
+This support enables you to do the following:
+
+* The Jira Create step can now be used to create issues with existing issues as their parent.
+* The Jira Update step can be used to update the parent of a specific issue.
+* A sub-task can be created using the Jira Create step.
+* A ticket parent can be provided by simply typing the parent issue key in **Parent Link** setting, such as `TJI-47890`. 
+
+To use issue links, do the following:
```
Comment (naman-goenka) [inline docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-jira-issues-in-cd-stages.md:115]: To use `Parent` field

**Pull request merged**